docs/content.zh/docs/dev/datastream/sinks.md

```diff
@@ -93,36 +93,36 @@ The `Sink` needs to be serializable. All configuration should be validated eager
 
 ### Custom sink topology
 
-For advanced developers, they may want to specify their own sink operator topology(A structure composed of a series of operators), such as collecting `committables` to one subtask and processing them together, or performing operations such as merging small files after `Committer`. Flink provides the following interfaces to allow expert users to customize the sink operator topology.
+For advanced developers and expert users, they may want to specify their own sink operator topology (A structure composed of a series of operators), such as collecting `committables` to one subtask and processing them together, or performing operations such as merging small files after `Committer`. Flink provides the following interfaces to allow expert users to customize the sink operator topology.
```

Comment:
I am curious what the difference is between advanced developers and expert users. If we want to agree and define on personas / roles for Flink then we could refer to them separately (if they are sufficiently different) - otherwise I suggest we stick to one way of referring to them (I am happy with either, but if it requires coding then developer in the role name would make sense)

Reply:
I think this is mainly aimed at developers because it is a specific interface.
But I think this place doesn't need to be so strict, because users may also have the need to understand this part.

Reply:
Personally I prefer specific words to prevent confusion, but am ok to leave as is if you have strong feelings on this. If we do this then we should refer to both in the sentence "allow expert users to ". Or it reads that this is not applicable to advanced developers